### DISCUSSION TITLE
Introducing repo.securityonion.net in 2.3.50 for CentOS users

### DISCUSSION THREAD

**Discussion by TOoSmOotH** [Security-Onion-Solutions/securityonion#3838] [General]
Starting in 2.3.50, we will be moving all CentOS installs to point to repo.securityonion.net for package downloads. This repo will contain all the necessary repos including epel, docker, saltstack, wazuh, etc. This should improve the experience for those users who have strict external connectivity requirements. We have also updated the FAQ with all of the external URLs an install uses once you update to 2.3.50.

https://docs.securityonion.net/en/2.3/faq.html#what-connectivity-does-security-onion-need-to-stay-up-to-date 

If you restrict access you will need to open repo.securityonion.net before you run soup when 2.3.50 is released.

**Comment by TheRealPancakes** [comment]
This is awesome news.

**Comment by smallboy69** [comment]
does the repo have security onion 2 application as i have installed ubuntu 20.4 and currently looking for the installation files for security onion 2 ...any version...

thanks

**Reply by dougburks** [reply]
If you want to install on Ubuntu, please follow the instructions in the documentation:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos

If you have further questions or problems, please start a new discussion rather than replying to discussions from 2021.
